c2corg_ui/views/route.py

```diff
@@ -46,3 +46,8 @@ def edit(self):
             'route_id': id
         })
         return self.template_input
+
+    @view_config(route_name=_API_ROUTE + '_history',
```

Comment:
Why using _API_ROUTE in the route_name whereas it has not been used for other route_names?
I think it is unnecessarly confusing, I would prefer we use the explicit route_name here.